src/python/ensembl/io/genomio/annotation/load.py

```diff
@@ -0,0 +1,180 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Load functional annotation from a file into a core database.
+"""
+
+import logging
+from pathlib import Path
+from typing import Any, Dict, List, Tuple
+
+from ensembl.core.models import Gene, Transcript, ObjectXref, Xref
+from ensembl.database import DBConnection
+from ensembl.io.genomio.utils import get_json
+from ensembl.utils.argparse import ArgumentParser
+from ensembl.utils.logging import init_logging_with_args
+from sqlalchemy.orm import Session
+
+
+FEAT_TABLE = {
+    "gene": "gene",
+    "mobile_element": "gene",
+    "transcript": "transcript",
+}
+
+
+def get_core_data(session: Session, table: str) -> Dict[str, Tuple[str, str, str]]:
+    """Load descriptions from a core."""
+
+    if table == "gene":
+        stmt = (
+            session.query(Gene.gene_id, Gene.stable_id, Gene.description, Xref.dbprimary_acc)
+            .select_from(Gene)
+            .outerjoin(ObjectXref, Gene.gene_id == ObjectXref.ensembl_id)
+            .outerjoin(Xref)
+            .where(ObjectXref.ensembl_object_type == "gene")
+        )
+    elif table == "transcript":
+        stmt = (
+            session.query(
+                Transcript.transcript_id, Transcript.stable_id, Transcript.description, Xref.dbprimary_acc
+            )
+            .select_from(Transcript)
+            .outerjoin(ObjectXref, Transcript.transcript_id == ObjectXref.ensembl_id)
+            .outerjoin(Xref)
+            .where(ObjectXref.ensembl_object_type == "transcript")
+        )
+
+    feat_data = {}
+    for row in session.execute(stmt):
+        (feat_id, stable_id, desc, xref_name) = row
+        feat_struct = (feat_id, stable_id, desc)
+        feat_data[stable_id] = feat_struct
+        if xref_name:
+            feat_data[xref_name] = feat_struct
+
+    return feat_data
+
+
+def load_descriptions(
+    session: Session, func_file: Path, report: bool = False, do_update: bool = False
+) -> None:
+    """Load gene and transcript descriptions in a core database."""
+    func = get_json(func_file)
+    logging.info(f"{len(func)} annotations from {func_file}")
+    tables_to_lookup = ("gene", "transcript")
+    for table in tables_to_lookup:
+        logging.info(f"Checking {table} descriptions")
+        feat_func = [feat for feat in func if feat["object_type"] == table]
+        logging.info(f"{len(feat_func)} {table} annotations from {func_file}")
+        feat_data = get_core_data(session, table)
+        logging.info(f"Loaded {len(feat_data)} {table} data")
+
+        stats = {
+            "not_supported": 0,
+            "not_found": 0,
+            "differs": 0,
+            "same": 0,
+            "no_new": 0,
```

Comment:
```suggestion
            "not_new": 0,
```
To follow the same nomenclature as the other keys.